src/briefcase/integrations/xcode.py

```diff
@@ -227,21 +224,19 @@ def ensure_xcode_is_installed(
                 preamble = """\
 Xcode appears to be installed, but the active developer directory is the Xcode
 command line tools. To make Xcode the active developer directory, run:
+
+    $ sudo xcode-select --switch /Applications/Xcode.app
 """
             else:
                 preamble = """\
 You have the Xcode command line tools installed; however, Briefcase requires
-a full Xcode install. Xcode can be downloaded from the macOS App Store.
-
-Once you have installed Xcode, you can make it the active developer directory
-by running:
+a full Xcode install. Xcode can be downloaded from the macOS App Store at
+<https://apps.apple.com/au/app/xcode/id497799835?mt=12>.
 """
```

Comment:
After installing Xcode, it becomes the active developer directory automatically, so the `xcode-select` command is not required in this case.